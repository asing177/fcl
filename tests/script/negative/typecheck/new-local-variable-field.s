@initial
go() {
  c.f = 1; // don't make `c` a new local variable
  terminate();
}
