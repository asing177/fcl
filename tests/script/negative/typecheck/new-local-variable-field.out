Error at Line 3, Column 3:
   Unbound variable:  c